cmd/nodecmd/validateSubnet.go

```diff
@@ -190,7 +190,7 @@ func validateSubnet(_ *cobra.Command, args []string) error {
 			}
 			continue
 		}
-		addedNodeAsPrimaryNetworkValidator, err := addNodeAsPrimaryNetworkValidator(nodeID, models.Fuji, i)
+		addedNodeAsPrimaryNetworkValidator, err := addNodeAsPrimaryNetworkValidator(nodeID, models.Fuji, i, host)
```

Comment:
try to pass the instance id, not the ansible host id, so as to be independent of ansible here